Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -142,8 +147,8 @@ After loading the script, a window that contains two sections, ``SCHEMA`` (top)
   Default: ``False``.
 - ``config_options_ptc``: Options to be passed to the command-line PTC pipetask. They will overwrite the values in ``cpPtc.yaml``.
   Default: ``-c isr:doCrosstalk=False``.
-- ``do_gain_from_flat_pairs``: Should the gain be estimated from each pair of flats taken at the same exposure time? Runs the ``cpPtc.yaml#generateGainFromFlatPair`` pipeline. Since this pipeline is a subset of the PTC pipeline, you can use use the ``config_options_ptc`` parameter to pass options to the ``ISR`` (Instrument Signature Removal) and ``cpExtract`` tasks which form this pipeline.
-  Default: ``False``.
+- ``do_gain_from_flat_pairs``: Should the gain be estimated from each pair of flats taken at the same exposure time? Runs the ``cpPtc.yaml#generateGainFromFlatPair`` pipeline. Since this pipeline is a subset of the PTC pipeline, you can use use the ``config_options_ptc`` parameter to pass options to the ``ISR`` (Instrument Signature Removal) and ``cpExtract`` tasks which form this pipeline. If ``True``, the script mode should be ``BIAS_DARK_FLAT``.
```

Comment:
Does your script evaluates all these conditions before running anything?

Reply:
Yes. In this case, if the parameter is true but the mode is not "BIAS_DARK_FLAT", nothing happens (i.e., the gain is not estimated from pairs of flats, because you need flats): https://github.com/lsst-ts/ts_externalscripts/blob/develop/python/lsst/ts/externalscripts/base_make_calibrations.py#L1611-L1617